README.md

```diff
@@ -27,3 +27,24 @@ atoms_ab = pyconvert(AbstractSystem, atoms_ase)
 newatoms_ase = convert_ase(atoms_ab)
 newatoms_ase.pop(4)
 ```
+
+### AtomsCalculators interface
+
+You can use ASE calculators in julia, by wrapping them to a `ASEcalculator` structure. Here is a brief example
```

Comment:
I think `AseCalculator` is the better capitalisation (in line with what `AtomsIO` does (e.g. `AseParser`).

Reply:
Package name is `ASEconvert` so being consistent with that is `ASEcalculator`

Reply:
ok fair enough.